ts/reply-queue.ts

```diff
@@ -115,17 +115,24 @@ function handleStreamReply(msg: amqp.Message, id: string) {
     contentType: 'application/json',
     persistent: false
   };
-  backpressure = !streamHandler.push(obj);
 
   if (stopped[id]) {
     options.channel.sendToQueue(msg.properties.replyTo, Buffer.from(JSON.stringify(Queue.STOP_STREAM_MESSAGE)), properties);
-    streamHandler.destroy();
+    streamHandler.push(null);
     delete options[id];
     delete stopped[id];
     delete streamHandlers[id];
+    logger.info(
```

Comment:
Can you also add 1-2 loggers in queue.ts?

Reply:
Done